### PR TITLE
docs: show --tty usage

### LINKED ISSUE #414: document limactl start --tty flag usage
Thank you for publishing this tool! I am quite excited to leverage it to improve my team's software development workflow.

I am prototyping a software development environment using Lima. 

I would love to have an option to override the default, interactive mode of `limactl start my-system.yaml`.

## implementation ideas / questions
- I expect this to be opt in behavior to not disturb the current userbase 
- what is a sensible flag name? I dislike `-f` because it may confuse people thinking that refers to the positional argument of a configuration file

### PR DESCRIPTION
**Pull request by efx**
Closes #414

### CODE REVIEW

**Comment by jandubois** [comment]
I kind of disagree that this is a prominent issue that needs to be shown explicitly before anything else.

The very next paragraph explains that `limactl start` will open an editor, and `limactl start --help` explains how to disable that behaviour:

```console
$ limactl start --help
Start an instance of Lima. If the instance does not exist, open an editor for creating new one, with name "default"

Usage:
  limactl start NAME|FILE.yaml|URL [flags]

Flags:
  -h, --help   help for start
      --tty    enable TUI interactions such as opening an editor, defaults to true when stdout is a terminal (default true)

Global Flags:
      --debug   debug mode
```

So if anything, I would support expanding the following paragraph from `limactl start <INSTANCE>` to `limactl start [--tty=false] <INSTANCE>` and adjusting the explanation accordingly.

File: README.md [hunk 1]
```diff
@@ -135,18 +135,13 @@ INFO[0029] READY. Run `lima` to open the shell.
...
-```
-
 Detailed usage:
 
-- Run `limactl start <INSTANCE>` to start the Linux instance.
+- Run `limactl start <INSTANCE> [--tty=false]` to start the Linux instance.
   The default instance name is "default".
   Lima automatically opens an editor (`vi`) for reviewing and modifying the configuration.
   Wait until "READY" to be printed on the host terminal.
+  `--tty=false` disables the interactive prompt.
```
Comment (jandubois) [inline README.md:144]: We should change `Lima automatically opens an editor` to `Lima offers to open an editor` above, and then write:

```suggestion
  `--tty=false` disables the interactive prompt to open an editor.
```

To make things more explicit for somebody just reading the `README` who hasn't actually run `limactl` before.

**Review by jandubois — APPROVED** [review]
Thanks, LGTM.

Leaving for @AkihiroSuda for more feedback, or merging.

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**